src/main/kotlin/skytils/skytilsmod/utils/ItemRarity.kt

```diff
@@ -26,28 +26,28 @@ import java.awt.Color
  * https://github.com/SteveKunG/SkyBlockcatia/blob/1.8.9/LICENSE.md
  * @author SteveKunG
  */
-enum class ItemRarity(val rarityName: String, val baseColor: EnumChatFormatting, val color: Color) {
```

Comment:
not really sure what this "rarityInt" is supposed to be for

Reply:
It's just a way to easily access the rarity without having to use .equals() all the time when comparing rarities (without it you would have to use itemRarity.rarityName.equals(blah blah blah) but instead you can have itemRarity.rarityInt == blah. It's also useful for using operators to compare what should be displayed on the griffin tracker, because with Strings there's no way to compare the relative rarity of items. If there's a better way to do it that I'm unaware of then that's fine with me ¯\_(ツ)_/¯

Reply:
you can't do ItemRarity == ItemRarity?

Reply:
Still, how would one compare values with operators?